src/cloud/project/project-enable-mfa-enforcement.md

```diff
@@ -34,9 +34,6 @@ magento-cloud ssh-cert:load
 
 The `ssh-cert:load` command generates the SSH certificate and installs it in the SSH agent of the local user.
 
-{:.bs-callout-tip}
```

Comment:
Did you remove this on purpose?

Reply:
Yes, it is inaccurate as written.  Even with an api token generate, you still need to generate an SSH certificate.